R/mmrm-methods.R

```diff
@@ -226,3 +226,11 @@ print.summary.mmrm <- function(x,
   cat("\n")
   invisible(x)
 }
+
+#' @importFrom generics tidy glance augment
+#' @export
+generics::tidy
+#' @export
+generics::glance
+#' @export
+generics::augment
```

Comment:
Ah, I would only have this here if you merge `tidiers.R` content into `mmrm-methods.R` completely, otherwise better to keep it in `tidiers.R`